bolt/lib/Passes/PointerAuthCFIAnalyzer.cpp

```diff
@@ -47,6 +47,7 @@ bool PointerAuthCFIAnalyzer::runOnFunction(BinaryFunction &BF) {
         // Not all functions have .cfi_negate_ra_state in them. But if one does,
         // we expect psign/pauth instructions to have the hasNegateRAState
         // annotation.
+        std::lock_guard<std::mutex> Lock(IgnoreMutex);
```

Comment:
Consistency nit: the other uses of this put the lock/ignore below the log message and next to the return.